website/docs/r/hdinsight_hbase_cluster.html.markdown

```diff
@@ -291,6 +293,32 @@ A `monitor` block supports the following:
 
 * `primary_key` - (Required) The Operations Management Suite (OMS) workspace key.
 
+---
+
+An `autoscale` block supports the following:
+
+* `recurrence` - (Optional) A `recurrence` block as defined below.
+
+-> **NOTE:** Capacity based autoscaling isn't supported to HBase clusters.
```

Comment:
we can remove this and conditionally make the `recurrence` block required here?

Reply:
I made option required. However, purpose of the Note is to emphasis on a difference between HBase and other cluster types.